.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
This is pretty awesome! But [pre-commit-hooks](https://github.com/pre-commit/pre-commit-hooks) says that this hook is deprecated and points to [upstream flake8](https://gitlab.com/pycqa/flake8). Can you update `repo` to the new one? Optionally, leave [using-hooks](https://flake8.pycqa.org/en/latest/user/using-hooks.html) link for anyone who wants to modify this later.

Reply:
Sure, I can add the black hook to this as well and see about updating the flake8 one.